synonyms.tsv

```diff
@@ -1363,3 +1363,4 @@ CONSO00352	Mini-Mental_Status_Exam	ncit:C74982	EXACT
 CONSO00353	1-(5-acetylamino-5-carboxypentyl)-3-hydroxy-5-hydroxymethyl-pyridinium	pmid:12784645	EXACT
 CONSO00353	Glyceraldehyde derived Pyridinium compound	pmid:12784645	EXACT
 CONSO00353	Glyceraldehyde-derived Pyridinium compound	pmid:12784645	EXACT
+CONSO00363	N-(6-(2-amino-4-fluorophenylamino)-6-oxohexyl)-4-methylbenzamide	ctd	EXACT
```

Comment:
Not a CTD identifier. That's an IUPAC name. Put in the synonyms file

Reply:
CTD doesn't issue identifiers for molecules. They're all references in MeSH

Reply:
@cthoyt please check if this is how you wanted the entry in synonyms file.